tools/platformio/platformio-build.py

```diff
@@ -146,7 +146,7 @@ def configure_application_offset(mcu, upload_protocol):
     env.Append(LINKFLAGS=["-Wl,--defsym=LD_FLASH_OFFSET=%s" % hex(offset)])
 
 
-if any(mcu in board_config.get("build.cpu") for mcu in ("cortex-m4", "cortex-m7")):
+if any(mcu in board_config.get("build.cpu") for mcu in ("cortex-m4", "cortex-m7")) and 'stm32wl' not in board_config.get('build.mcu'):
```

Comment:
Just to be consistent with the rest of the script, please use double quotes(`"`). Also, there is already `mcu` variable available, so no need to retrieve it again via `board_config.get('build.mcu')`.

Reply:
Hi @valeros yeah, you're totally right. I actually had to turn my auto linting off because of line lengths and stuff, so it slipped through the cracks. Fixed it now.